content/master/concepts/packages.md

```diff
@@ -34,24 +34,23 @@ the {{<hover line="6" label="install">}}spec.package{{</hover>}} value to the
 location of the configuration package.
 
 {{< hint "important" >}}
-Beginning with Crossplane version 1.15.0 Crossplane uses the Upbound Marketplace
-Crossplane package registry at `xpkg.upbound.io` by default for downloading and
+Beginning with Crossplane version 1.20.0 Crossplane uses the [crossplane-contrib](https://github.com/orgs/crossplane-contrib/packages) GitHub Container Registry at `xpkg.crossplane.io` by default for downloading and
 installing packages. 
 
 Specify the full domain name with the `package` or change the default Crossplane
 registry with the `--registry` flag on the [Crossplane pod]({{<ref "./pods">}})
 {{< /hint >}}
 
 For example to install the 
-[Upbound AWS reference platform](https://marketplace.upbound.io/configurations/upbound/platform-ref-aws/v0.6.0).
+[Getting Started Configuration](https://github.com/crossplane-contrib/configuration-getting-started), 
```

Comment:
this `crossplane-contrib/configuration-getting-started ` configuration package repo needs to be created/ported, it doesn't seem to exist yet 🤓

Reply:
Yes, that needs creating before this PR gets merged.